my-app/src/components/Body/Content/BlogView/BlogView.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import {Button} from "@material-ui/core";
+import useStyles from "./style";
+import {Typography} from "@material-ui/core";
+import Pagination from '@material-ui/lab/Pagination';
+import arrow from "../../../../assets/img/arrow-up-circle.svg";
+import PenToolWhite from "../../../../icons/PenToolWhite";
+
+const createData = (name, posted, status, stats, views, dots) => {
+    return {name, posted, status, stats, views, dots};
+};
+
+const rows = [
+    createData('Design: A Survival Guide for Beginners', 'Posted 3 days ago', "Scheduled", 120, "views", ""),
+    createData('Design: A Survival Guide for Beginners', 'Posted 3 days ago', 'Draft', 120, "views", ""),
+    createData('Design: A Survival Guide for Beginners', 'Posted 3 days ago', 'Scheduled', 120, "views", ""),
+    createData('Design: A Survival Guide for Beginners', 'Posted 3 days ago', 'Published', 0, "views", ""),
+    createData('Design: A Survival Guide for Beginners', 'Posted 3 days ago', 'Published', 120, "views", ""),
+];
```

Comment:
useless function

if you want to create mock data - replace it to separate file and describe there

Reply:
done